alu_u32/src/lt/columns.rs

```diff
@@ -13,17 +13,26 @@ pub struct Lt32Cols<T> {
     pub byte_flag: [T; 4],
 
     /// Bit decomposition of 256 + input_1 - input_2
-    pub bits: [T; 10],
```

Comment:
The 10th bit of `256 + input_1 - input_2` will always be 0 when `input_1, input_2` are 8-bit numbers. Is there a reason to include a 10th column here?